pkg/scaffold/internal/templates/v2/controller/controller_suitetest.go

```diff
@@ -56,6 +58,51 @@ func (f *SuiteTest) Validate() error {
 	return f.Resource.Validate()
 }
 
+const (
+	importMarker    = "imports"
+	addSchemeMarker = "scheme"
+)
+
+// GetMarkers implements file.Inserter
+func (f *SuiteTest) GetMarkers() []file.Marker {
+	return []file.Marker{
+		file.NewMarkerFor(f.Path, importMarker),
+		file.NewMarkerFor(f.Path, addSchemeMarker),
+	}
+}
+
+const (
+	apiImportCodeFragment = `%s "%s"
+`
+	addschemeCodeFragment = `err = %s.AddToScheme(scheme.Scheme)
+Expect(err).NotTo(HaveOccurred())
+
+`
+)
+
+// GetCodeFragments implements file.Inserter
+func (f *SuiteTest) GetCodeFragments() file.CodeFragmentsMap {
+	fragments := make(file.CodeFragmentsMap, 2)
+
+	// Generate import code fragments
+	imports := make([]string, 0)
+	imports = append(imports, fmt.Sprintf(apiImportCodeFragment, f.Resource.ImportAlias, f.Resource.Package))
```

Comment:
`imports := []string{fmt.Sprintf(...)}`?

Reply:
I did it this way on purpose, to allow easy addition and deletion of code fragments. Do you have a strong opinion on this?